decidim-core/app/cells/decidim/content_blocks/participatory_space_social_networks_cell.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentBlocks
+    class ParticipatorySpaceSocialNetworksCell < BaseCell
+      def show
+        return if social_handler_items.blank?
+
+        render
+      end
+
+      def social_handler_items
+        Decidim::Assembly::SOCIAL_HANDLERS.filter_map do |handler|
```

Comment:
This needs to be refactored ... If i will not enable the assemblies, this would fail.